packages/paste-theme/src/themeProvider.tsx

```diff
@@ -22,10 +23,11 @@ function getProviderThemeProps(theme: ThemeVariants, customBreakpoints?: string[
         ...SendGridTheme,
         breakpoints: customBreakpoints || SendGridTheme.breakpoints,
       };
-    case ThemeVariants.CONSOLE:
+    case DeprecatedThemeVariants.CONSOLE:
+      isDeprecatedTheme(DeprecatedThemeVariants.CONSOLE);
       return {
-        ...ConsoleTheme,
-        breakpoints: customBreakpoints || ConsoleTheme.breakpoints,
+        ...DefaultTheme,
+        breakpoints: customBreakpoints || DefaultTheme.breakpoints,
```

Comment:
If you remove these lines, it will call the function and fall into the default case (since there's no return or break)

Reply:
Yeah... I tried to do that but typescript said it expected a return. I think it is because I have  line 27 where I run `isDeprecatedTheme`